cogboard-app/src/main/kotlin/com/cognifide/cogboard/ssh/auth/SSHAuthData.kt

```diff
@@ -27,10 +29,19 @@ class SSHAuthData(private val config: JsonObject) {
 
     private fun hasAuthTypeCorrectCredentials(authType: AuthenticationType): Boolean =
             when {
-                authType == SSH_KEY && key.isNotBlank() -> true
+                authType == SSH_KEY && key.isNotBlank() -> {
+                    prepareForSSHKeyUsage()
```

Comment:
Nazwa funkcji sugeruje tylko sprawdzanie warunku logicznego i zwrócenie true/false. Tutaj z kolei mamy zaszytą jakąś logikę ZAPISYWANIA czegoś do pliku. Code smell - koniecznie do zmiany

Reply:
Poprawmy to proszę

Reply:
Funkcja z przygotowaniem do użycia klucza SSH poszła do konstruktora